bundle/config/mutator/python/python_diagnostics.go

```diff
@@ -0,0 +1,121 @@
+package python
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"regexp"
+	"strconv"
+
+	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/cli/libs/dyn"
+)
+
+type pythonDiagnostic struct {
```

Comment:
I did research on how other tools report warnings:
- go vet
- eslint
- pylint
- terraform validate

All k8s tools report the path as a string. I will change it to do the same.

Reply:
Also note that this format is purely internal, and we wouldn't show this to users, occasionally, we might need to read it for debugging.

If we ever want CLI to report warnings in a structured format, it can use any format not necessarily the same as here.

Reply:
+1 -- I wouldn't want to couple them as they might need to evolve independently.